### ISSUE TITLE
Checks awaiting first synchronization

### ISSUE THREAD

**Issue by Jonathan-Garber** [wh1te909/tacticalrmm#268]
I've got one agent stuck on "checks awaiting first synchronization"
V 0.4.6 Agent V 1.4.3

I've uninstalled and re-installed this agent but it still gets stuck here. Any pointers?

**Comment by wh1te909** [comment]
open cmd as admin and run
 ```"C:\Program Files\TacticalAgent\tacticalrmm.exe" -m runchecks -log debug -logto stdout```
will print alot of debug info should see the error there

**Comment by Jonathan-Garber** [comment]
There was no error there at all. But it still says awaiting first synchronization

```
2021/02/05 17:23:23.572323 DEBUG RESTY
==============================================================================
~~~ REQUEST ~~~
GET  /api/v3/blahblahblah/checkrunner/  HTTP/1.1
HOST   : api.blahblahblah
HEADERS:
        Accept: application/json
        Authorization: Token blahblahblah
        Content-Type: application/json
        User-Agent: go-resty/2.4.0 (https://github.com/go-resty/resty)
BODY   :
***** NO CONTENT *****
------------------------------------------------------------------------------
~~~ RESPONSE ~~~
STATUS       : 200 OK
PROTO        : HTTP/1.1
RECEIVED AT  : 2021-02-05T17:23:23.5723231-05:00
TIME DURATION: 135.998ms
HEADERS      :
        Allow: GET, PATCH, HEAD, OPTIONS
        Content-Length: 614
        Content-Type: application/json
        Date: Fri, 05 Feb 2021 22:22:04 GMT
        Referrer-Policy: same-origin
        Server: nginx
        Vary: Origin
        X-Content-Type-Options: nosniff
        X-Frame-Options: DENY
BODY         :
{
   "agent": 12,
   "check_interval": 120,
   "checks": [
      {
         "id": 3,
         "assigned_tasks": [
            {
               "id": 3,
               "enabled": true
            }
         ],
         "script": null,
         "check_type": "eventlog",
         "status": "pending",
         "threshold": null,
         "disk": null,
         "ip": null,
         "script_args": [],
         "timeout": null,
         "svc_name": null,
         "pass_if_start_pending": null,
         "pass_if_svc_not_exist": false,
         "restart_if_stopped": null,
         "log_name": "Application",
         "event_id": 15006,
         "event_id_is_wildcard": false,
         "event_type": "ERROR",
         "event_source": "MSExchangeTransport",
         "event_message": "Exchange Transport service is rejecting messages because available disk space is below",
         "fail_when": "contains",
         "search_last_days": 1,
         "agent": 12
      }
   ]
}
==============================================================================
```

**Comment by wh1te909** [comment]
[tacticalrmm.zip](https://github.com/wh1te909/tacticalrmm/files/5935717/tacticalrmm.zip)

please try same command with this exe. Don't need to replace the exe in program files, just put this exe wherever u want but call it same way with `-m runchecks -log debug -logto stdout`
this one should print some more debug info and then paste output please

**Comment by Jonathan-Garber** [comment]
```
tacticalrmm.exe -m runchecks -log debug -logto stdout
```
That had the exact same output as above. There was no additional output.

**Comment by wh1te909** [comment]
lol sorry im retarded. bug in event log check with latest agent xD i'll have a new release out shortly with the fix

**Comment by Jonathan-Garber** [comment]
LOL - no problem. I'll update when I can. Thanks.

**Comment by wh1te909** [comment]
[tacticalrmm.zip](https://github.com/wh1te909/tacticalrmm/files/5935780/tacticalrmm.zip)
can you try please with this exe just as a sanity check before i release :)

**Comment by Jonathan-Garber** [comment]
Definitely way more output there and now the interface says checks are passing. So it appears to be fixed in that version of the exe. Looks like it properly searched the event logs that time.

**Comment by wh1te909** [comment]
thanks, released in 1.4.4
